modules/openapi-generator/src/main/resources/typescript-fetch/modelOneOf.mustache

```diff
@@ -36,6 +36,8 @@ export function {{classname}}FromJSONTyped(json: any, ignoreDiscriminator: boole
         return {{{.}}}FromJSONTyped(json, true);
     }
     {{/oneOf}}
+
+    return json;
```

Comment:
**thought (non-blocking):** 
Here, the fallback case returns the raw `json`.  But a similar fallback in the `xxxToJSON` (below) returns `{}`.

I don't think it makes a huge practical difference (if this happens for real, one should go and fix the code to be compatible with the spec, instead of finding smart ways around it).

Just looks a bit inconsistent.

Reply:
Indeed. I change it to `return {} as any`.

(I needed to add type-assertion because `{}` is not compatible with the return type.
But `json` is also `any`. There is no big difference.)